### PR TITLE
*: hide the system variables `tidb_track_aggregate_memory_usage`

### PR DESCRIPTION
**Pull request by wshwsh12**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

How it Works:
1. Remove the system variable `tidb_track_aggregate_memory_usage`.
2. Track memory usage of aggregate executor by default.

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:

### Check List 

Tests 

- Unit test
- Integration test

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 

- Tracking memory usage of Aggregate function GA.

### CODE REVIEW

File: session/bootstrap.go [hunk 1]
```diff
@@ -1448,6 +1451,13 @@ func upgradeToVer65(s Session, ver int64) {
 	doReentrantDDL(s, CreateStatsFMSketchTable)
 }
 
+func upgradeToVer66(s Session, ver int64) {
+	if ver >= version66 {
+		return
+	}
+	mustExecute(s, "set @@global.tidb_track_aggregate_memory_usage = 1")
```
Comment (XuHuaiyu) [inline session/bootstrap.go:1458]: Add a test case for this.
Check the value in version65 and version66

File: session/bootstrap.go [hunk 2]
```diff
@@ -474,9 +474,11 @@ const (
 	version64 = 64
 	// version65 add mysql.stats_fm_sketch table.
 	version65 = 65
+	// version66 open the feature `track_aggregate_memory_usage` by default.
```
Comment (ichn-hu) [inline session/bootstrap.go:477]: s/open/enable

**Review by ichn-hu — CHANGES_REQUESTED** [review]
Rest LGTM

File: session/bootstrap.go (hunk 2 above)
Comment (wshwsh12) [reply session/bootstrap.go]: Addressed.

**Review by XuHuaiyu — COMMENTED** [review]
LGTM

**Review by ichn-hu — APPROVED** [review]

**Pull request merged**